open-ce/validate_config.py

```diff
@@ -46,19 +46,19 @@ def validate_env_config(conda_build_config, env_config_files, variants, reposito
                                                variant['cudatoolkit'],
                                                repository_folder=repository_folder,
                                                conda_build_config=conda_build_config)
-                validate_build_tree(recipes, variant)
+                validate_build_tree(recipes, recipes.get_external_dependencies(variant))
```

Comment:
Do we still need to call validated_build_tree here? It seems like we are calling it twice now. Once in the BuildTree constructor, and once here.

Reply:
You're right. I didn't commit the change that removed this.
Thanks for catching that!

Reply:
Actually, this is the call for the validate_config entry point.
The build_tree constructor calls the validate build tree function.

Reply:
It looks like BuildTree's constructor will call `validate_config.validate_build_tree`. The constructor is called right before this line. This line then calls `validate_config.validate_build_tree` again.

Reply:
The tests seem to be taking twice as long as before as well.

Reply:
So the entry_point path was doing a double call now that the validation is happening as part of the build_tree constructor. That should be fixed now.